lib/update-bridgehead.sh

```diff
@@ -55,7 +55,7 @@ for DIR in /etc/bridgehead $(pwd); do
     OUT=$(retry 5 git -C $DIR fetch 2>&1 && retry 5 git -C $DIR pull 2>&1)
   else
     log "INFO" "Git is using proxy ${HTTP_PROXY_URL} from ${CONFFILE}"
-    OUT=$(retry 5 git -c http.proxy=$HTTP_PROXY_URL -c https.proxy=$HTTPS_PROXY_URL -C $DIR fetch 2>&1 && retry 5 git -c http.proxy=$HTTP_PROXY_URL -c https.proxy=$HTTPS_PROXY_URL -C $DIR pull 2>&1)
+    OUT=$(retry 5 git -c http.proxy=$PROXY -c https.proxy=$PROXY -C $DIR fetch 2>&1 && retry 5 git -c http.proxy=$PROXY -c https.proxy=$PROXY -C $DIR pull 2>&1)
```

Comment:
You are mixing http and https proxy. This will lead to https calls beeing send to the http proxy, which could matter if the sites have two different proxies for that. I think we will need to have the code above create the PROXY variable for http and a SECURE_PROXY variable for https